### DISCUSSION TITLE
http_proxy in dev mode

### DISCUSSION THREAD

**Discussion by filojalo** [apache/camel-karavan#1434] [Q&A] [answered]
### Ask a question

Hi, I am doing a PoC using Karavan to generate a Camel route. I am running a docker container on a linux VM. Due to company limitations we do not have network access to the internet, but we are able to use a http_proxy to do so.
I was able to get the docker images themselves, however when I try to run in dev mode I am unable to configure any http_proxy to be used by the ghcr.io/apache/camel-karavan-devmode image, so I cannot get the maven dependecies.
Is there a way to get around it? Thanks.

### Variant

Web Application

### Container Management (if applicable)

Docker

### Operating System (if applicable)

Linux

### Version

4.8.0

**Comment by mgubaidullin** [comment, marked as answer]
There is no proxy configuration in DevMode container. You need to build custom DevMode container and configure Karavan to use your custom dev container

**Comment by mgubaidullin** [comment]
To configure Karavan to use custom DevMode image set env variable KARAVAN_DEVMODE_IMAGE
